vakt/storage/sql/model.py

```diff
@@ -0,0 +1,109 @@
+import json
+
+from sqlalchemy import Column, Integer, String, ForeignKey
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+
+from ...policy import Policy
+
+Base = declarative_base()
+
+
+class PolicySubjectModel(Base):
+    """Storage model for policy subjects"""
+
+    __tablename__ = 'policy_subject'
```

Comment:
Generally, it's a good practice to name tables in plural.

One more thing: since vakt can be integrated with user's custom code (an thus existing databases) tables need to be a little bit more scoped: say,  `vakt_policy_subjects`. Ideally there needs to be an ability to provide custom table names for greater flexibility.

Reply:
Yes, had intended to change the names but it skipped my mind. In regards to the ability for custom table names, the approach in this [link](https://stackoverflow.com/questions/19163911/dynamically-setting-tablename-for-sharding-in-sqlalchemy) can be used.